cmd/osquery-perf/agent.go

```diff
@@ -1692,6 +1693,23 @@ func (a *agent) softwareMacOS() []map[string]string {
 		uniqueSoftware = uniqueSoftware[:a.softwareCount.unique-a.softwareCount.uniqueSoftwareUninstallCount]
 	}
 
+	// Software with Duplicate Bundle identifiers
+	duplicateCount := (a.softwareCount.common * a.softwareCount.duplicateBundleIdentifiersPercent) / 100
+	duplicateBundleSoftware := make([]map[string]string, duplicateCount)
+	groupSize := 4
+	for i := 0; i < duplicateCount; i++ {
+		bundleIDIndex := i / groupSize
+		bundleID := fmt.Sprintf("com.fleetdm.osquery-perf.common_%d", bundleIDIndex%a.softwareCount.common)
+
+		duplicateBundleSoftware[i] = map[string]string{
+			"name":              fmt.Sprintf("RENAMED_DuplicateBundle_%d", i),
```

Comment:
We discussed making this configurable via CLI param to make testing easier in load test env